### ISSUE TITLE
ESLint error in Vue components

### ISSUE THREAD

**Issue by xon52** [xon52/x5-gmaps#47]
After testing the latest version of this in a mockup Typescript project, I have found that inside a Vue component

I cannot seem to avoid the eslint error: `'google' is not defined.eslint (no-undef)`.
It seems to work fine inside a `.ts` file, and all the Google API Types are available (e.g. `google.maps.LatLngLiteral`), but that error persists.

Would love some help in finding what's needed here. It may even be an eslint problem, but not sure.

![](https://cdn.discordapp.com/attachments/608306024306311171/825615408845029376/unknown.png)

**Comment by xon52** [comment]
This problem doesn't exist using the older dev-dependencies:
```js
    "@typescript-eslint/eslint-plugin": "^2.33.0",
    "@typescript-eslint/parser": "^2.33.0",
```
But these are quite old.

An alternative I could try is making my own namespace which can handle all the input type definitions and so we don't need the google definitions anymore - but not today ⌚

**Comment by kbottner** [comment]
Does your .eslintrc.js file contain the google line in the extends section?

```
extends: [
    "eslint:recommended",
    "plugin:import/errors",
    "plugin:import/warnings",
    "plugin:import/typescript",
    "google",
    "plugin:@typescript-eslint/recommended"
  ],

```

**Comment by xon52** [comment]
Can't seem to get that to work @kbottner ?
Are you using the latest? Is it working for you?

**Comment by kbottner** [comment]
I filed that addition to issue #25 and haven't moved back to X5 since I was waiting on a resolution. But I am using Google Maps directly and that is how I got eslint to appropriately work.

**Comment by xon52** [comment]
Fixed this in #62 I think with a typeRoot property in tsconfig.
